### PR TITLE
Use rmt-server as a prefix for systemd units 212

### PR DESCRIPTION
**Pull request by kalabiyau**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          71 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 19 | Lint/Syntax: %if: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 19 | Lint/Syntax: unexpected token tSTRING_CONTENT<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 20 | Lint/Syntax: %define: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 21 | Lint/Syntax: %endif: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 23 | Lint/Syntax: ambiguous first argument; put parentheses or a space even after the operator<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 23 | Lint/Syntax: unknown regexp options: har<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 24 | Lint/Syntax: %define: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 24 | Lint/Syntax: unexpected token error<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 25 | Lint/Syntax: %define: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 26 | Lint/Syntax: %define: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 26 | Lint/Syntax: unexpected token error<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 27 | Lint/Syntax: %define: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 29 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 30 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 30 | Lint/Syntax: no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)</digit></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 31 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 32 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 32 | Lint/Syntax: unexpected token tNL<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 33 | Lint/Syntax: unexpected token kOR<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 34 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 35 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 35 | Lint/Syntax: unknown regexp options: ftwar<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 37 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 37 | Lint/Syntax: %ix: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 43 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 44 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 45 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 46 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 47 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 48 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 49 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 50 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 51 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 52 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 53 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 55 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 55 | Lint/Syntax: unexpected token kIN<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 56 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 56 | Lint/Syntax: unexpected token kIN<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 58 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 59 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 60 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 61 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 62 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 63 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 64 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 65 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 65 | Lint/Syntax: no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)</digit></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 66 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 66 | Lint/Syntax: no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)</digit></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 67 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 67 | Lint/Syntax: no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)</digit></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 68 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 70 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 71 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 72 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 72 | Lint/Syntax: no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)</digit></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 73 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 73 | Lint/Syntax: no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)</digit></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 74 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 75 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 76 | Lint/Syntax: unexpected token tCOLON<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 78 | Lint/Syntax: %description: unknown type of percent-literal<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 80 | Lint/Syntax: unexpected token tCONSTANT<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 82 | Lint/Syntax: unexpected token tIDENTIFIER<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 149 | Lint/Syntax: unexpected token tLABEL<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 150 | Lint/Syntax: unexpected token tIDENTIFIER<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 158 | Lint/Syntax: bare backslash only allowed before newline<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 158 | Lint/Syntax: unexpected token tIDENTIFIER<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 159 | Lint/Syntax: unexpected token tIDENTIFIER<br />
(Using Ruby 2.3 parser; configure using <code>TargetRubyVersion</code> parameter, under <code>AllCops</code>)</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">package/rmt-server.spec | 159 | Lint/Syntax: unexpected <code>@'
(Using Ruby 2.3 parser; configure using </code>TargetRubyVersion<code> parameter, under </code>AllCops`)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ikapelyukhin** [comment]
@kalabiyau You broke `Danger` :grimacing:

**Comment by lagartoflojo** [comment]
@kalabiyau hehe I'll check

**Comment by kalabiyau** [comment]
@lagartoflojo thanks a lot!

**Comment by tmuntaner** [comment]
@kalabiyau Could you please also make a changelog entry for this change?

**Comment by lagartoflojo** [comment]
I made `danger` only comment about adding a changelog line if there's changes in the `app` directory. Not sure if we need changelog entries otherwise tbh. Not sure if this PR would really merit a changelog entry, for example.

**Comment by tmuntaner** [comment]
@lagartoflojo  We're renaming service files that users interact with, how else would they be alerted?
@kalabiyau You missed a `rcrmt-migration` on line 138
```
ln -fs %{_sbindir}/service %{buildroot}%{_sbindir}/rcrmt-server-migration
```

**Comment by lagartoflojo** [comment]
If we renamed these after 1.0, then yeah, a changelog entry would be needed. OTOH, I don't think we'd rename these files after 1.0.

File: package/rmt-server.spec [hunk 1]
```diff
@@ -135,7 +135,7 @@ install -m 444 %{SOURCE10} %{buildroot}%{_unitdir}
 install -m 444 %{SOURCE11} %{buildroot}%{_unitdir}
 mkdir -p %{buildroot}%{_sbindir}
 ln -fs %{_sbindir}/service %{buildroot}%{_sbindir}/rcrmt
```
Comment (ikapelyukhin) [inline package/rmt-server.spec]: This as well, then? `rcrmt-server`?

Comment (tmuntaner) [reply package/rmt-server.spec]: Thanks for catching that

**Comment by tmuntaner** [comment]
@lagartoflojo This is probably ready for review now. I'm just running some checks with the package.

File: support/rmt [hunk 2]
```diff
@@ -34,7 +34,7 @@ validate_rpm_if_installed() {
 
 RPMLIST=(rmt-server nginx mariadb)
 
-STATUS_UNITS=(rmt.service rmt-server-sync.timer rmt-server-mirror.timer mariadb.service nginx.service)
+STATUS_UNITS=(rmt-server.service rmt-server-migrate.service rmt-server-sync.timer rmt-server-mirror.timer mariadb.service nginx.service)
```
Comment (lagartoflojo) [inline support/rmt]: it's `rmt-server-migration.service`

**Pull request merged**